sdks/python/src/test_server_sdk/install.py

```diff
@@ -155,17 +155,3 @@ def install_binary(bin_dir: Path):
         print(f"An error occurred during binary installation: {e}")
         sys.exit(1)
 
-
```

Comment:
Removed the customized build hook since we don't want to include the binary into the built wheel.